alert-common/src/main/java/com/synopsys/integration/alert/common/rest/ResponseFactory.java

```diff
@@ -167,4 +168,13 @@ public static ResponseStatusException createNotImplementedException(@Nullable St
         return new ResponseEntity<>(header, HttpStatus.FOUND);
     }
 
+    public <T> ResponseEntity<T> createResponseFromAction(ActionResult<T> actionResult) {
```

Comment:
Would it be better to give ActionResult a method that would convert itself into a ResponseEntity? At least these basic conversions? Or is that making ActionResult do too much?

Reply:
I think that would be making ActionResult do too much.  We may want to convert it differently.